.travis.yml

```diff
@@ -11,9 +11,10 @@ env:
 install:
   - bundle install
```

Comment:
Is this line still needed?

Reply:
I thought so, but I'm kinda figuring it out as I go :D